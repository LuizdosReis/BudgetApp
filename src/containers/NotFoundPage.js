import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
  <div className="content">
    not found -
    {' '}
    <Link to="/">Go to Home</Link>
  </div>
);

export default NotFoundPage;
