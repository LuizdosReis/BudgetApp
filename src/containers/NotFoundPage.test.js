import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFoundPage from './NotFoundPage';

test('should render NotFoundPage correctly', () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
