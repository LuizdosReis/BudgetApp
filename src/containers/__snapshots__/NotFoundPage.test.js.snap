// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage correctly 1`] = `
<div
  className="content"
>
  not found -
   
  <Link
    replace={false}
    to="/"
  >
    Go to Home
  </Link>
</div>
`;
